Docker/FreshRSS.Apache.conf

```diff
@@ -20,15 +36,32 @@ TraceEnable Off
 	AllowOverride None
 	Include /var/www/FreshRSS/p/.htaccess
 	Options FollowSymLinks
-	Require all granted
+
+	<IfDefine !OIDC_ENABLED>
+		Require all granted
+	</IfDefine>
+
+	<IfDefine OIDC_ENABLED>
+		AuthType openid-connect
+		Require valid-user
+	</IfDefine>
```

Comment:
The root is not supposed to be password protected but rather public. Only `/i/` should be restricted